packages/common/src/hooks/useListData/useListData.ts

```diff
@@ -1,10 +1,11 @@
+import { ObjectWithStringKeys } from './../../types/utility';
 import { SortBy } from './../useSortBy/useSortBy';
 import { UseMutateFunction } from 'react-query';
 import { useQueryClient, useMutation, useQuery } from 'react-query';
 import { QueryParams, useQueryParams } from '../useQueryParams';
 import { SortRule } from '../useSortBy';
 
-export interface ListApi<T> {
+export interface ListApi<T extends ObjectWithStringKeys> {
```

Comment:
ive used T a bunch and I don't actually like it but I also wasn't sure if `SomeEntity` or some other name was more useful 🤷